contrib/tig-completion.bash

```diff
@@ -29,6 +29,27 @@
 #       at source time then all lookups will be done on demand,
 #       which may be slightly slower.
 
+#tig-completion requires __git_complete
+#* If not defined, source git completions script so __git_complete is available
+if ! declare -f __git_complete &>/dev/null; then
+  _bash_completion=$(pkg-config --variable=completionsdir bash-completion 2>/dev/null) ||
```

Comment:
Nitpick: the indentation should be consistent with the rest of the file.

Reply:
Thank you. I will fix that.